packages/eslint-plugin/src/rules/no-misused-spread.ts

```diff
@@ -67,7 +115,13 @@ export default createRule<Options, MessageIds>({
 
   defaultOptions: [
     {
+      allowStrings: false,
+      allowPromises: false,
+      allowFunctions: false,
+      allowMaps: false,
```

Comment:
My 2c: I think that we don't want `allowPromises` or `allowMaps`.
Like the Array case - these cases are special cases that are always going to be bad.

Reply:
gotcha, done